pkg/virt-api/webhooks/validating-webhook/validating-webhook.go

```diff
@@ -1336,6 +1336,19 @@ func ValidateVirtualMachineInstanceSpec(field *k8sfield.Path, spec *v1.VirtualMa
 	return causes
 }
 
+func ValidateVirtualMachineInstanceMetadata(field *k8sfield.Path, vmi *v1.VirtualMachineInstance) []metav1.StatusCause {
+	var causes []metav1.StatusCause
+	// Validate ignition feature gate if set when the corresponding annotation is found
+	if vmi.GetObjectMeta().GetAnnotations()["kubevirt.io/ignitiondata"] != "" && !virtconfig.IgnitionEnabled() {
```

Comment:
can we make a const value for kubevirt.io/ignitiondata. we have a place somewhere in the code where labels and annotations are defined.

Reply:
done